tests/bluetooth/bsim_bt/bsim_test_eatt_notif/src/client_test.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Can you elaborate a bit on this test? 

It seems that it connects multiple EATT and then sends a large number of notifications, but then also starts disconnecting and reconnecting EATT and then attempting to send a notification during discovery. 

What is the purpose of this test?

Reply:
Yes, this test is meant to increase the coverage for the notifications over EATT as they should be reliable.
But you can propose more ideas on how to better test the notifications if you think there is not sufficient coverage here.

Reply:
Sorry, I see that my comment was unclear - Can you add information about the test as comments in the file, so that may be more clear in the future what it is testing and why? 

This test is a little less straight forward than most other tests, as it does multiple things to test EATT notifications :)